program/src/processor.rs

```diff
@@ -65,9 +67,27 @@ pub fn process_instruction(
 ) -> ProgramResult {
     let instruction_type = decode_instruction_type(input)?;
     let account_info_iter = &mut accounts.iter();
-    let accounts = SlashingAccounts::new(account_info_iter)?;
     match instruction_type {
+        SlashingInstruction::CloseViolationReport => {
+            let report_account = next_account_info(account_info_iter)?;
+            let destination_account = next_account_info(account_info_iter)?;
+            let system_program_account = next_account_info(account_info_iter)?;
+
+            if !check_id(report_account.owner) || report_account.data_is_empty() {
```

Comment:
i didn't check the version here as it should be fine to close older or uninitialized reports
once we have multiple versions we will have code to convert between them in order to read the destination/epoch fields and such